### PR TITLE
#1456: Add Malayalam flair embeddings models

### PR DESCRIPTION
**Pull request by aiswaryas-qb**
To load use 'ml-forward' (Forward Model) and 'ml-backward' (Backward Model).

### CODE REVIEW

File: flair/embeddings.py [hunk 1]
```diff
@@ -1934,6 +1934,9 @@ def __init__(self, model, fine_tune: bool = False, chars_per_chunk: int = 512):
             # Japanese
             "ja-forward": f"{aws_path}/embeddings-v0.4.1/lm__char-forward__ja-wikipedia-3GB/japanese-forward.pt",
             "ja-backward": f"{aws_path}/embeddings-v0.4.1/lm__char-backward__ja-wikipedia-3GB/japanese-backward.pt",
+            # Malayalam
+            "ml-forward": f"https://raw.githubusercontent.com/qburst/models-repository/master/FlairMalayalamModels/ml-forward.pt",
+            "ml-backward": f" https://raw.githubusercontent.com/qburst/models-repository/master/FlairMalayalamModels/ml-backward.pt",
```
Comment (alanakbik) [inline flair/embeddings.py]: The backward model does not load for me due to the whitespace before the URL. Could you change: 

`             "ml-backward": f" https://raw.githubusercontent.com/qburst/models-repository/master/FlairMalayalamModels/ml-backward.pt", `

to 
`
             "ml-backward": f"https://raw.githubusercontent.com/qburst/models-repository/master/FlairMalayalamModels/ml-backward.pt", `

**Comment by alanakbik** [comment]
@aiswaryas-qb thanks and please excuse the delay in responding! Could you remove the whitespace in the download URL for the backward model as indicated in the comment? Then everything works: 

```python
# example sentence
sentence = Sentence('ഈ പർവത പാത മനോഹരമായ തടാകത്തിനടുത്തേക്ക് കയറിച്ചെല്ലുന്നു')

# init Malayalam embeddings
embeddings = StackedEmbeddings(
    [FlairEmbeddings('ml-forward'),
     FlairEmbeddings('ml-backward')]
)

# embed sentence
embeddings.embed(sentence)

# check embeddings
for token in sentence:
    print(token)
    print(token.embedding[:10])
```

**Comment by aiswaryas-qb** [comment]
Hi @alanakbik I have removed the backspace and have updated embeddings.py file. Thank you.

**Comment by alanakbik** [comment]
Great, thanks for adding this!

**Pull request merged**